src/main/java/in/ac/bits/protocolanalyzer/persistence/repository/ElasticSearchFactoryImpl.java

```diff
@@ -0,0 +1,41 @@
+package in.ac.bits.protocolanalyzer.persistence.repository;
+
+import org.elasticsearch.common.settings.ImmutableSettings;
+import org.elasticsearch.node.NodeBuilder;
+import org.springframework.core.env.Environment;
+
+public class ElasticSearchFactoryImpl implements ElasticSearchFactory {
+	
+	@Override
+	public ImmutableSettings.Builder settingsBuilder(Environment env) {
+		String clusterName = env.getProperty("elasticsearch.cluster.name");
+        String nodeName = env.getProperty("elasticsearch.node.name");
+        String corsEnabled = env.getProperty("elasticsearch.http.cors.enabled");
+        String allowOrigin = env
+                .getProperty("elasticsearch.http.cors.allow-origin");
+        String allowMethods = env
+                .getProperty("elasticsearch.http.cors.allow-methods");
+        String allowHeaders = env
+                .getProperty("elasticsearch.http.cors.allow-headers");
+		String dataPath = env.getProperty("elasticsearch.path.data");
+		String logPath = env
+		.getProperty("elasticsearch.path.logs");
+		return ImmutableSettings
+                .settingsBuilder().put("cluster.name",clusterName)
```

Comment:
The methods in the method chain must align at one indentation.

Reply:
Corrected in the latest PR